src/qibo/quantum_info/superoperator_transformations.py

```diff
@@ -303,6 +303,50 @@ def to_chi(
     return channel
 
 
+def to_stinespring(
+    channel,
+    partition: Optional[Union[List[int], Tuple[int, ...]]] = None,
+    nqubits: Optional[int] = None,
+    initial_state_env=None,
+    backend=None,
+):
+    """Converts quantum ``channel`` :math:`U` to its Stinespring representation :math:`U_{0}`.
+
+    It uses the Kraus representation as an intermediate step.
+
+    Args:
+        channel (ndarray): quantum channel.
+        nqubits (int, optional): total number of qubits in the system that is
+            interacting with the environment. Must be equal or greater than
+            the number of qubits ``kraus_ops`` acts on. If ``None``,
```

Comment:
Here with `kraus_op` you mean the input `channel`? I yes, like it seems to me from the following implementation, I would explicitly refer here to `channel`.